yb_migrate/src/migration/oracle.go

```diff
@@ -291,8 +291,10 @@ func OracleGetAllTableNames(source *utils.Source) []string {
 	defer db.Close()
 
 	var tableNames []string
-	query := fmt.Sprintf("SELECT table_name FROM all_tables WHERE owner = '%s' "+
-		"AND TEMPORARY = 'N' AND table_name NOT LIKE 'DR$%%' ORDER BY table_name ASC", source.Schema)
+	query := fmt.Sprintf(`SELECT table_name FROM all_tables WHERE owner = '%s' 
+		AND TEMPORARY = 'N' AND table_name NOT LIKE 'DR$%%' AND (owner, table_name) not in 
+		(select owner, mview_name from all_mviews union all select log_owner, log_table from all_mview_logs)
+		ORDER BY table_name ASC`, source.Schema)
```

Comment:
Consider properly formatting the query for easy readability:
```
SELECT table_name
FROM all_tables
WHERE owner = '%s' AND TEMPORARY = 'N' AND table_name NOT LIKE 'DR$%%' AND
  (owner, table_name) not in (
    SELECT owner, mview_name
    FROM all_mviews
    UNION ALL
    SELECT log_owner, log_table
    FROM all_mview_logs)
ORDER BY table_name ASC
```
Add a comment explaining the query to help readers new to Oracle.

Reply:
sure, will do that